addons/io/org.openhab.io.javasound/README.md

```diff
@@ -0,0 +1,4 @@
+# Default audio sink and source
+
+This IO bundle provides a Java API based `AudioSink` and `AudioSource`
+to let a user play and capture audio on the host system.
```

Comment:
Second line needed here?

Reply:
Not really. It doesn't hurt though ^^

Reply:
We usually say one line per sentence.
In my view it is inconsistent to tell people splitting their texts up and then use multiple lines for one sentence on the other hand.

The generated view will be the same anyway, so its just my opinion.

Reply:
I'll quickly change it. Will just not be signed off then anymore.